be/src/formats/parquet/encoding_dict.h

```diff
@@ -203,22 +205,75 @@ class DictDecoder<Slice> final : public Decoder {
         return Status::OK();
     }
 
-    Status get_dict_values(const std::vector<int32_t>& dict_codes, Column* column) override {
+    Status get_dict_values(const std::vector<int32_t>& dict_codes, const NullableColumn& nulls,
+                           Column* column) override {
+        const std::vector<uint8_t>& null_data = nulls.immutable_null_column_data();
+        bool has_null = nulls.has_null();
+        bool all_null = false;
+
+        if (has_null) {
+            size_t count = SIMD::count_nonzero(null_data);
```

Comment:
do you think it's better to split it into 64 per batch? otherwise, only a few null value will lead to the most expensive branch

Reply:
It's already handled in 64items per batch way.

Reply:
I mean handle it in 64 batch in LOC:221~LOC:231

Reply:
Would that be long & verbose?